packages/playwright-core/src/server/webkit/wkBrowser.ts

```diff
@@ -361,4 +362,11 @@ export class WKBrowserContext extends BrowserContext {
   async cancelDownload(uuid: string) {
     await this._browser._browserSession.send('Playwright.cancelDownload', { uuid });
   }
+
+  _validateEmulatedViewport(viewportSize?: types.Size | null) {
+    if (!viewportSize)
+      return;
+    if (process.platform === 'win32' && this._browser.options.headful && (viewportSize.width < 250 || viewportSize.height < 250))
+      throw new Error(`WebKit on Windows has a minimal viewport of 250x250.`);
```

Comment:
let's make sure that this numbers are consistent with what we have in Firefox

Reply:
There is none.